packages/grid/_modules_/grid/models/gridOptions.tsx

```diff
@@ -404,6 +405,15 @@ export interface GridOptions {
    * You can find all the translation keys supported in [the source](https://github.com/mui-org/material-ui-x/blob/HEAD/packages/grid/_modules_/grid/constants/localeTextConstants.ts) in the GitHub repository.
    */
   localeText: Partial<GridLocaleText>;
+  /**
+   * Callback fired when scrolling to the bottom of the grid viewport.
+   * @param param
+   */
+  onRowsScrollEnd?: (params: GridRowScrollEndParams) => void;
+  /**
+   * Set the area at the bottom of the grid viewport where onRowsScrollEnd is called.
+   */
+  scrollEndThreshold: number;
```

Comment:
If we could sort them asc, it would be 👌

Reply:
Do you mean all of the props? Because now none of them are really sorted.

Reply:
I would propose we sort them all. The value would be in matching what the docs will generate, they are sorted to be easier to navigate. It's not a huge value, but I guess if we do it, only for new code, the cost isn't high either.

Reply:
It took a while but they are alphabetically sorted now.